CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ### Unreleased
+  - Added SEFT tab
```

Comment:
Where is the change for this?

Reply:
It's already in.  It got forgotten about when it was originally put in.
After this PR goes in, I'm going to take a tag for this and call it 2.0.0 as we've been fairly irresponsible with the changelog for this repo

Reply:
Woah thats a lot of changes. i hadn't expanded the view